frontend/src/store/modules/activity.ts

```diff
@@ -87,24 +112,44 @@ export const useActivityStore = defineStore("activity", {
       this.setActivityListForUser({ userId, activityList });
       return activityList;
     },
-    async fetchActivityList(params: {
+    async fetchPagedActivityList(params: {
       typePrefix: string | string[];
       container?: number | string;
       order: "ASC" | "DESC";
       user?: number;
       limit?: number;
       level?: string | string[];
+      token?: string;
     }) {
       const url = `/api/activity?${stringify(params, {
         arrayFormat: "repeat",
       })}`;
-      const response = (await axios.get(url)).data;
-      const activityList: Activity[] = response.data.map(
-        (activity: ResourceObject) => {
-          return convert(activity, response.included);
-        }
+      const responseData = (await axios.get(url)).data;
+      const activityList = convertEntityList(
+        responseData,
+        "activityList",
+        convert,
+        getActivityFromIncludedList
       );
-      return activityList;
+      const nextToken = isPagedResponse(responseData, "activityList")
+        ? responseData.data.attributes.nextToken
+        : "";
+      return {
+        nextToken,
+        activityList,
+      };
+    },
+    async fetchActivityList(params: {
+      typePrefix: string | string[];
+      container?: number | string;
+      order: "ASC" | "DESC";
```

Comment:
Looks like we're using `DESC` to handle the pagination, but what if users choose `ASC` as the order? (Can users do this?)

Reply:
Currently, sorting by users is not supported.